### PR TITLE
Writing for split-toning module standardized

### PR DESCRIPTION
**Pull request by mepi0011**
The writing of Split Toning adapted to the usual writing.
See also pull request # 3695

### CODE REVIEW

File: src/iop/splittoning.c [hunk 1]
```diff
@@ -87,7 +87,7 @@ typedef enum dt_iop_splittoning_picker_t
 
 const char *name()
 {
-  return _("split toning");
+  return _("Split Toning");
```
Comment (TurboGit) [inline src/iop/splittoning.c]: All word in dt are lower case. So we cannot change that.

File: src/iop/colorbalance.c [hunk 2]
```diff
@@ -272,10 +272,10 @@ static void add_preset(dt_iop_module_so_t *self, const char *name,
 void init_presets(dt_iop_module_so_t *self)
 {
   // these blobs were exported as dtstyle and copied from there:
-  add_preset(self, _("split-toning teal-orange (2nd instance)"),
+  add_preset(self, _("Split Toning teal-orange (2nd instance)"),
```
Comment (TurboGit) [inline src/iop/colorbalance.c]: I would probably standardize with "split-toning".

Changing the case here is not possible or we need to change all the case of all dt strings.

File: src/iop/splittoning.c (hunk 1 above)
Comment (mepi0011) [reply src/iop/splittoning.c]: I didn't know this rule.

Changing to lower case will no problem, but then I would prefer to write it split-toning as you mentioned below.

I would use the same spelling for the user manual and adjust it accordingly.

File: src/iop/colorbalance.c (hunk 2 above)
Comment (mepi0011) [reply src/iop/colorbalance.c]: See comment above.

File: src/iop/splittoning.c (hunk 1 above)
Comment (TurboGit) [reply src/iop/splittoning.c]: Fine with me.

Comment (mepi0011) [reply src/iop/splittoning.c]: Done!

**Comment by TurboGit** [comment]
Thanks !

**Comment by TurboGit** [comment]
Merged on master and 3.0.1.

**Pull request merged**